### DISCUSSION TITLE
Swift syntax support?

### DISCUSSION THREAD

**Discussion by subdigital** [zed-industries/zed#6625] [Feature Requests]
When can we expect support for Swift syntax (both colors and LSP)?

**Comment by subdigital** [comment]
Ah, I see this [has been requested](https://github.com/zed-industries/community/issues/355) already. Eager to see this land!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
